homeassistant/components/huawei_lte/__init__.py

```diff
@@ -158,16 +165,18 @@ def _setup_lte(hass, lte_config) -> None:
             mode = "ip"
     except ValueError:
         mode = "hostname"
-    mac = get_mac_address(**{mode: host})
+    func = partial(get_mac_address, **{mode: host})
+    mac = await hass.async_add_executor_job(func)
 
-    connection = AuthorizedConnection(url, username=username, password=password)
+    func = partial(AuthorizedConnection, url, username=username, password=password)
+    connection = await hass.async_add_job(func)
```

Comment:
Please replace all use of `hass.async_add_job` here with either `hass.async_add_executor_job` or `hass.async_create_task` depending on the target function.